client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/unified-design-picker.tsx

```diff
@@ -82,10 +81,6 @@ const UnifiedDesignPickerStep: Step = ( { navigation, flow } ) => {
 
 	// ********** Logic for fetching designs
 	const selectStarterDesigns = ( allDesigns: StarterDesigns ) => {
-		if ( isEnabled( 'signup/design-picker-style-selection' ) ) {
```

Comment:
With this, we should remove this `signup/design-picker-style-selection` everywhere, treating it as `true`.

Reply:
Should we create a new PR for the clean-up? Happy to do it in this PR as well.

Reply:
Your call but if I were you I'll do it in the same PR because they are related and we don't need another round of testing for that other PR 🙈

Reply:
Fair point!